src/Util.ts

```diff
@@ -2,9 +2,45 @@ import BigNumber from 'bignumber.js';
 import * as Web3 from 'web3';
 import * as Web3WsProvider from 'web3-providers-ws';
 
-import { IBlock, ITxRequest } from './Types';
+import { IBlock, IService, IServices, ITxRequest } from './Types';
+
+function apiCall(url: string): Promise<any> {
+  const { window } = this;
+  return window.fetch(url)
+    .then((response :any) => {
+      if (!response.ok) {
+        throw new Error(response.statusText)
+      }
+      return response.json()
+    })
+    .then((data: any) => { /* <-- data inferred as { data: T }*/
+      return data.data
+    })
+}
+
+const SERVICES: IServices = {//should return GWEI value of the gasPrice
+  '1': [
```

Comment:
"1" is for network id I guess. Maybe introduce an enum to represent those networks?